enginetest/queries/queries.go

```diff
@@ -7500,6 +7500,28 @@ SELECT * FROM my_cte;`,
 			{4},
 		},
 	},
+	{
```

Comment:
It looks like all the tests we run are testing against a string type. You should add a test that calls `find_in_set` on an enum column and on a set column so that we have coverage over those cases, too.